store/tikv/snapshot.go

```diff
@@ -311,14 +312,18 @@ func extractLockFromKeyErr(keyErr *pb.KeyError) (*Lock, error) {
 }
 
 func extractKeyErr(keyErr *pb.KeyError) error {
+	failpoint.Inject("ErrMockRetryableOnly", func(val failpoint.Value) {
```

Comment:
Add this to mock conflict == nil, but retryable != nil, e.g. error: txnLockNotFound.